### PR TITLE
Styling homepage

### PR DESCRIPTION
**Pull request by cryptokoala4**
Minor changes in css for homepage look (font, transparency, text shadows)

### CODE REVIEW

File: app/assets/stylesheets/users.scss [hunk 1]
```diff
@@ -1,3 +0,0 @@
-// Place all the styles related to the users controller here.
-// They will automatically be included in application.css.
```
Comment (dshcheung) [inline app/assets/stylesheets/users.scss:2]: delete this file if you don't need it

File: app/views/pages/index.html.erb [hunk 2]
```diff
@@ -1,17 +1,15 @@
...
-        </div>
+  <div class="row">
+    <h2 class="home-header2">Love takeout sushi? You're in the right place!</h2>
+    <h1 class="home-header">Sushi-ME Away!</h1>
+    <div class="col-xs-12 col-sm-6 col-sm-offset-3 col-md-4 col-md-offset-4 well transparency">
+      <div>
+        <%= form_tag 'restaurants/search' do |f| %>
+           <div class="form-group">
+              <%= collection_select(:restaurant, :district_id, District.order(:name).all, :id, :name, {prompt: "Select a distict"}, {class: "form-control", style: "color: black"}) %>
+          </div>
```
Comment (dshcheung) [inline app/views/pages/index.html.erb:10]: indentations

File: app/views/restaurants/index.html.erb [hunk 3]
```diff
@@ -1,19 +1,31 @@
...
+<div class="row">
+  <% @restaurants.each do |restaurant| %>
+      <div class="col-xs-12 col-md-6 col-lg-4 restaurant-cards">
+        <%= link_to restaurant do %>
+          <div class="card-wrapper thumbnail transparency">
+            <h4><%= restaurant.name %></h4>
+            <div class="address">
+              <%= restaurant.address %>
+            </div>
+            <div>
```
Comment (dshcheung) [inline app/views/restaurants/index.html.erb:19]: indentations

**Comment by dshcheung** [comment]
add a new line to the end of page for the files that needs it

File: app/assets/stylesheets/users.scss (hunk 1 above)
Comment (cryptokoala4) [inline app/assets/stylesheets/users.scss:2]: amended

File: app/views/pages/index.html.erb (hunk 2 above)
Comment (cryptokoala4) [inline app/views/pages/index.html.erb:10]: amended

File: app/views/restaurants/index.html.erb (hunk 3 above)
Comment (cryptokoala4) [inline app/views/restaurants/index.html.erb:19]: amended

**Comment by cryptokoala4** [comment]
Fixed all!

**Pull request merged**